ui.apps/src/main/content/jcr_root/apps/valtech/aecu/config/org.apache.sling.jcr.repoinit.RepositoryInitializer~setacls.config

```diff
@@ -0,0 +1,23 @@
+scripts=["
+    create path /var/groovyconsole/scripts/aecu(nt:folder)
+
+	## TODO: index?
```

Comment:
Can you also add the index here? You deleted the original index definition in your PR.

Reply:
I just moved the index definition. The todo is a leftover. All service users and ACLs are now created in this repo iiniit script.

Reply:
Removed comment in https://github.com/valtech/aem-easy-content-upgrade/pull/150/commits/05668365e6575469a28eba6ae2b3c367cdf52fad